sdcm/cluster_azure.py

```diff
@@ -10,19 +10,21 @@
 # See LICENSE for more details.
 #
 # Copyright (c) 2022 ScyllaDB
-
+import json
 import logging
 from functools import cached_property
 from typing import Dict, List
 
 from sdcm import cluster
 from sdcm.provision.azure.provisioner import AzureProvisioner
 from sdcm.provision.provisioner import PricingModel, VmInstance
+from sdcm.sct_events.system import SpotTerminationEvent
 from sdcm.sct_provision import region_definition_builder
 from sdcm.sct_provision.instances_provider import provision_instances_with_fallback
 from sdcm.utils.decorators import retrying
 
 LOGGER = logging.getLogger(__name__)
+SPOT_TERMINATION_CHECK_OVERHEAD = 15
```

Comment:
overhead ? why not call it timeout, as the other implementations does. it's not exactly clear what overhead mean in this context

Reply:
:+1:

Reply:
to me sounds much better